x-pack/plugins/apm/public/components/app/mobile/service_overview/geo_map/map_layers/get_http_requests_map_layer_list.ts

```diff
@@ -48,7 +47,10 @@ const label = i18n.translate(
   }
 );
 
-export async function getHttpRequestsLayerList(maps?: MapsStartApi) {
+export async function getHttpRequestsLayerList(
+  maps: MapsStartApi | undefined,
```

Comment:
What's the difference between declaring `maps?: ...` and `maps: MapsStartApi | undefined`?